moto/s3/models.py

```diff
@@ -2228,6 +2228,13 @@ def put_object_acl(
         # TODO: Support the XML-based ACL format
         if key is not None:
             key.set_acl(acl)
+            bucket = self.get_bucket(key.bucket_name)  # type: ignore
+
+            notify_event_name = (
+                notifications.S3NotificationEvent.OBJECT_ACL_UPDATE_EVENT
+            )
+
+            notifications.send_event(self.account_id, notify_event_name, bucket, key)
```

Comment:
The `put_object_acl`-method is also called internally when completing a multipart upload, and I don't think we want an ACL update notification in that scenario.

We had the same problem when implementing `put_object()`, and added a new parameter called `disable_notification`. I think we should have the same here, so that calling `complete_multipart_upload()` does not result in an ACL event.

https://github.com/getmoto/moto/blob/3eb9530c00802f7bb4478e1b5ac0d14e3fda21ce/moto/s3/models.py#L2202

Reply:
Thank you for the review!
I have fixed the issue so that the ACL event is no longer triggered when complete_multipart_upload() is called.

(https://github.com/getmoto/moto/pull/8349/commits/bba4e5dff38afe01bd19e44e16817077c46a2810 and https://github.com/getmoto/moto/pull/8349/commits/afb1e2f7aa8e116e84cde3b30140a400666d3c21)